mqtt/mqtt-bridge/src/controller/bridges.rs

```diff
@@ -58,12 +58,16 @@ impl Bridges {
     }
 
     pub(crate) async fn send_update(&mut self, update: BridgeUpdate) {
-        if let Some(config) = self.config_updaters.get_mut(update.name()) {
-            if let Err(e) = config.send_update(update).await {
-                error!("error sending bridge update {:?}", e);
+        let endpoint = update.endpoint().to_owned();
```

Comment:
why `to_owned()`?

Reply:
I need to have a copy of an endpoint that I can log in the `error!`, b/c `update` got consumed before that in `send_update()`.